src/commands/mod.rs

```diff
@@ -63,18 +66,24 @@ pub(crate) async fn register_commands(ctx: &Context) {
     );
 
     results.push(
-        Command::create_global_application_command(&ctx.http, |command| {
-            chad::register(command)
-        })
+        Command::create_global_application_command(&ctx.http, |command| {chad::register(command)})
+        .await,
+    );
+
+    results.push(
+        Command::create_global_application_command(&ctx.http, |command| {skull::register(command)})
             .await,
     );
 
-     results.push(
-            Command::create_global_application_command(&ctx.http, |command| {
-                skull::register(command)
-            })
+    if get_state(ctx).await.config.tenor_api_key.clone().is_some() {
+        //Register commands that use the Tenor API here!
+        results.push(
+            Command::create_global_application_command(&ctx.http, |command| {cat::register(command)})
                 .await,
         );
+    } else {
+        warn!("Warning: missing tenor_api_key, commands like cat that use the Tenor API won't work!");
```

Comment:
Just a final change 😄 sorry

```suggestion
        warn!("Missing tenor_api_key, commands like cat that use the Tenor API won't work!");
```